schemas/nuhs.json

```diff
@@ -101,27 +113,35 @@
                 "propertyOrder": 110,
                 "advanced": true
             },
+            "install_ntp": {
+                "type": "boolean",
+                "title": "Install ntp client on NUH",
+                "default": true,
+                "description": "Install ntp client on NUH. Defaults to true",
```

Comment:
You shouldn't say the default in the description because the default is usually added automatically when generating input forms.